app/views/pages/index.html.slim

```diff
@@ -0,0 +1,4 @@
+.container
+  /= render partial: 'pages/page_title', collection: @pages, as: :page
```

Comment:
output pages in table. with few columns (title, URL, type), mini version of screenshot preview

Reply:
Screenshot I will add later when will be added Carrierwave gem
that's ok?
![image](https://user-images.githubusercontent.com/32734554/34207360-cf814dd2-e592-11e7-823b-51b13a4f112d.png)

Reply:
OK